example/jwhois.conf

```diff
@@ -56,6 +56,11 @@ whois-servers {
 	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.verio.net";
 
 	#
+	# Catch ntt.net netblock records
+	#
+	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.gin.ntt.net";
```

Comment:
Well, it is really hard to read diff in the issue #3. Can you attach to an
issue as a file or email it to me?
On Jan 20, 2015 10:31 PM, "Jonas Öberg" <notifications@github.com> wrote:

> In example/jwhois.conf
> <https://github.com/jonasob/jwhois/pull/11#discussion-diff-23253032>:
>
> > @@ -56,6 +56,11 @@ whois-servers {
> >  	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.verio.net";
> >
> >  	#
> > +	# Catch ntt.net netblock records
> > +	#
> > +	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.gin.ntt.net";
>
> This seems as if it would match the same as the previous verio regexp. Are
> there any examples of an ntt netblock record? The regexp needs to be
> updated accordingly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jonasob/jwhois/pull/11/files#r23253032>.
>